smi2ass.py

```diff
@@ -27,15 +27,14 @@
 major = sys.version_info.major
 minor = sys.version_info.minor
 
-if major == 3 and minor == 7:
+if major == 3 and minor >= 7:
     import html
 elif major == 3 and minor < 7:
     from html.parser import HTMLParser
 else:
     print ('version 3.x needed') 
 
-default_lang_code = 'kor'
-default_font_name = 'sans-serif'
+default_font_name = 'Malgun Gothic'
```

Comment:
How about using fonts like "Noto Sans CJK KR" instead, given Malgun Gothic is not available in non-Windows?

Reply:
This is part of my mistake, while using and testing this program, I modified the ass to match with my subtitle settings.